examples/todo/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/bitfield/script"
+)
+
+func main() {
+	listPath := "."
+	if len(os.Args) > 1 {
+		listPath = os.Args[1]
+	}
+	// filter hidden directories and files
+	filterFiles := regexp.MustCompile(`^\..*|/\.`)
+	files := script.FindFiles(listPath).RejectRegexp(filterFiles)
+	content := files.EachLine(func(filePath string, builderFile *strings.Builder) {
+		p := script.File(filePath)
+		lineNumber := 1
+		p.EachLine(func(str string, build *strings.Builder) {
+			findTodo, err := hasTodo([]byte(str))
```

Comment:
Could we use `strings.Contains` here to simplify? This saves calling a function, building a regexp, and translating back and forth from `string` to `[]byte`.

Reply:
Yes, we could use strings.Contains to simplify the example further and for the purpose of the example, it would work.